freedomofpress/securedrop-client
# Add AppArmor profile for client and bump to securedrop-client 0.0.11

# Description

Towards https://github.com/freedomofpress/securedrop-workstation/issues/234

# Test Plan
- [ ] Ensure you are running 4.14.158 kernel in sd-svs (requires latest updates to sd-svs-buster-template)
- [ ] On this branch, build the tarball (`python3 setup.py dist`)
- Use this packaging branch: https://github.com/freedomofpress/securedrop-debian-packaging/pull/105 to build the 0.0.11 securedrop-client package (e.g.: `PKG_VERSION=0.0.11 PKG_PATH=../securedrop-client/dist/securedrop-client-0.0.11.tar.gz make securedrop-client`
- install apparmor-utils in sd-svs (or sd-svs-buster-template)
- move/install this deb into sd-svs (or sd-svs-buster-template)
- [ ] sudo aa-status should report securedrop-client is in enforce mode
- [ ] The client works as expected
- [ ] There are no apparmor violations in /var/log/syslog
- [ ] Changelog for 0.0.11 is accurate

# Checklist

If these changes modify code paths involving cryptography, the opening of files in VMs or network (via the RPC service) traffic, Qubes testing in the staging environment is required. For fine tuning of the graphical user interface, testing in any environment in Qubes is required. Please check as applicable:
